src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,13 +1,31 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne(fetch = FetchType.EAGER)
     private Country country;
 
     public Actor() {
     }
 
+    public Actor(String name, Country country) {
```

Comment:
where is the constructor with all params? where is toString method?

Reply:
not fixed